### PR TITLE
[TAAS-82] Add OCHA Services to vocabulary site

### PR DESCRIPTION
**Pull request by emmajane**
Adds a new vocabulary summary for OCHA Services menu items. Uses Andrej as the contact person.

### CODE REVIEW

**Comment by emmajane** [comment]
This should NOT be merged until https://github.com/UN-OCHA/taas-data/pull/78 is merged!!

File: _vocabs/ocha-services.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+title: Website Navigation - OCHA Services Menu Items
+display-title: Website Navigation - OCHA Services Menu Items (BETA)
+status: JSON available
+json-url: http://vocabulary.unocha.org/json/beta-v1/ocha_services.json
+gss-url: https://docs.google.com/spreadsheets/d/1DYclP1DfBN08vXoEQYK7J2Oo8MxllKTWjXBU-LHkeXE/edit#gid=856823080
+managed-by: OCHA Digital Services
+contact-email: verity@un.org
+---
+
+When implementing the OCHA common design, these websites should be listed in the header navigation under "Other OCHA Services".
```
Comment (lazysoundsystem) [inline _vocabs/ocha-services.md]: Looks good, but this line appears to be aimed only at OCHA developers - how about something like: "Key OCHA online services highlighted in the header of many OCHA sites." - but rephrased a bit better. 

(Do we have a clear criteria for which sites are included in this list? If we can publish it, here would be a good place to do it.)

Comment (emmajane) [reply _vocabs/ocha-services.md]: @lazysoundsystem great feedback, thanks! I've amended based on your new language. Let me know if that's better?

Criteria is going to be discussed at the GIFT meeting next week (previously it was based on number of visitors but this was "unfair" for new websites). Andrej was hoping to have something online for review / discussion next week to show how the JSON file could be made available.

Comment (lazysoundsystem) [reply _vocabs/ocha-services.md]: Yes, looks good to me, and we can update it when the criteria is agreed.

**Comment by emmajane** [comment]
@andrejverity Can you take a look and see if the language looks okay?

**Comment by andrejverity** [comment]
@emmajane only adjustment I would make is to point the email address to digitalservices@humanitarianresponse.info (rather than my personal email address). With that adjustment, let's get this live!

**Comment by emmajane** [comment]
@andrejverity adjustment made. I think we're good to go! I'll click the fancy merge button and see what happens next. :)

**Pull request merged**